package/system/procd/files/procd.sh

```diff
@@ -413,6 +413,33 @@ _procd_set_config_changed() {
 	ubus call service event "$(json_dump)"
 }
 
+procd_service_running() {
+	local service="$1"
+
+	[ -n "$service" ] || return 1
+
+	json_init
+	json_add_string name "$service"
+	local result="$(ubus call service list "$(json_dump)")"
+
+	# load & process result
+	json_load "$result"
+
+	# silence warnings from here on
+	local _json_no_warning=1
+	result=1
```

Comment:
dont reuse the result var please it makes the code hard to read. also drop the default value assignment.